doc/source/docs/user_guide/io.rst

```diff
@@ -265,5 +265,5 @@ the spatial information.
     https://github.com/geopandas/geo-arrow-spec
 
     This metadata specification does not yet make stability promises. As such,
-    we do not yet recommend using this in a production setting unless you are
```

Comment:
I think this perhaps should still be we - as it is we meaning geopandas developers rather than the conversational reader and the writer we in some of the other places.

Reply:
Thanks for taking a look, @m-richards. I considered not changing this, but the "we" felt ambiguous. Whereas "it is not recommend..." is more direct and still true. If the documentation says something is not recommended, the reader should understand that without having to figure out who recommended it. 

I can change it back if you'd like.

Reply:
Hi @Leviob, I would leave it as you've changed it for now. (Clearly I'm not impartial on the use of tense, but), I see the benefit not changing as I the style guide is easier to enforce the less exceptions there are. 

(for reference, I think the "who" actually can be important - as it dictates where a user should look for more information or follow up advice/ questions - but that can also done more directly without using "we", which as you say is a bit ambiguous. Here it is a moot point as this particular section of the documentation probably needs to be updated anyway xref #2693)